cli/src/commands/GenerateCommand.ts

```diff
@@ -136,15 +143,15 @@ export class GenerateCommand extends Command {
           return {
             ...config.from,
             relativePath: this.relativePath ?? config.from.relativePath,
-          };
+          } as FileOptions;
```

Comment:
I need to try this locally, I'm curious why this was working before and it's not working anymore 🤔

Reply:
casting to FileOptions in that case statement was optional.

The actual reason was this in the URL case statement:

<img width="921" alt="image" src="https://user-images.githubusercontent.com/271912/188711770-2b4504ba-327c-4633-a639-8416f20607da.png">

Reply:
Strange… on `main`:
![image](https://user-images.githubusercontent.com/1761469/188713748-2576847e-f29f-45d8-9bfa-6dbb7e88584b.png)

Something is fishy ^^

Reply:
Hmm, let me debug a lit more.

Reply:
Is your VSCode using correct typescript version?

Reply:
When I say to VSCode use workspace version it generates this. (maybe because I am on macOS)

<img width="1145" alt="image" src="https://user-images.githubusercontent.com/271912/188717179-e100cfbe-ef34-428b-89af-a06139a68b5a.png">

Reply:
Indeed, this is breaking with the newest version

![image](https://user-images.githubusercontent.com/1761469/188717797-c8aed6dd-5f57-4ac6-a5a3-d14a634800e6.png)

but this is the correct fix is here:

![image](https://user-images.githubusercontent.com/1761469/188718328-0ce7cf6d-2f8e-4c32-b4f6-e8330f65155c.png)

Reply:
(please note that I updated `typanion` and `clipanion` locally before trying this fix ^^) (https://github.com/arcanis/typanion/issues/9)

Reply:
Hmm, nice debugging! 

without bumping `typanion` `as const` still works.

What should we do in this case?

1. cast `as const` and remove casting on returning objects
2. bump typanion and clipanion  and also cast `as const`
3. ?

Reply:
I would say `1.` add the `as const` and remove the casting, this is safer 😃 If nothing break, since we are in bumping version, let's add `typanion` and `clipanion` 😅

Reply:
I applied the first solution.

I want to postpone typanion and clipanion version bump for now. We can create another PR later for it. Let's keep this PR as minimal as possible. I shouldn't even bump the Lerna in this PR.  (but I couldn't stop myself 😅)